skymp5-server/cpp/lib_espm/espm.h

```diff
@@ -779,6 +781,232 @@ class GMST : public RecordHeader
+  MagickaRate,
+  StaminaRate,
+  SpeedMult,
+  InventoryWeight,
+  CarryWeight,
+  CritChance,
+  MeleeDamage,
+  UnarmedDamage,
+  Mass,
+  VoicePoints,
+  VoiceRate,
+  DamageResist,
+  PoisonResist,
+  FireResist,
+  ElectricResist,
+  FrostResist,
+  MagicResist,
+  DiseaseResist,
+  PerceptionCondition,
+  EnduranceCondition,
+  LeftAttackCondition,
+  RightAttackCondition,
+  LeftMobilityCondition,
+  RightMobilityCondition,
+  BrainCondition,
+  Paralysis,
+  Invisibility,
+  NightEye,
+  DetectLifeRange,
+  WaterBreathing,
+  WaterWalking,
+  IgnoreCrippledLimbs,
+  Fame,
+  Infamy,
+  JumpingBonus,
+  WardPower,
+  RightItemCharge_or_EquippedItemCharge,
+  ArmorPerks,
+  ShieldPerks,
+  WardDeflection,
+  Variable01,
+  Variable02,
+  Variable03,
+  Variable04,
+  Variable05,
+  Variable06,
+  Variable07,
+  Variable08,
+  Variable09,
+  Variable10,
+  BowSpeedBonus,
+  FavorActive,
+  FavorsPerDay,
+  FavorsPerDayTimer,
+  LeftItemCharge_or_EquippedStaffCharge,
+  AbsorbChance,
+  Blindness,
+  WeaponSpeedMult,
+  ShoutRecoveryMult,
+  BowStaggerBonus,
+  Telekinesis,
+  FavorPointsBonus,
+  LastBribedIntimidated,
+  LastFlattered,
+  MovementNoiseMult,
+  BypassVendorStolenCheck,
+  BypassVendorKeywordCheck,
+  WaitingForPlayer,
+  OneHandedMod,
+  TwoHandedMod,
+  MarksmanMod,
+  BlockMod,
+  SmithingMod,
+  HeavyArmorMod,
+  LightArmorMod,
+  PickPocketMod,
+  LockpickingMod,
+  SneakMod,
+  AlchemyMod,
+  SpeechcraftMod,
+  AlterationMod,
+  ConjurationMod,
+  DestructionMod,
+  IllusionMod,
+  RestorationMod,
+  EnchantingMod,
+  OneHandedSkillAdvance,
+  TwoHandedSkillAdvance,
+  MarksmanSkillAdvance,
+  BlockSkillAdvance,
+  SmithingSkillAdvance,
+  HeavyArmorSkillAdvance,
+  LightArmorSkillAdvance,
+  PickPocketSkillAdvance,
+  LockpickingSkillAdvance,
+  SneakSkillAdvance,
+  AlchemySkillAdvance,
+  SpeechcraftSkillAdvance,
+  AlterationSkillAdvance,
+  ConjurationSkillAdvance,
+  DestructionSkillAdvance,
+  IllusionSkillAdvance,
+  RestorationSkillAdvance,
+  EnchantingSkillAdvance,
+  LeftWeaponSpeedMult,
+  DragonSouls,
+  CombatHealthRegenMult,
+  OneHandedPowerMod,
+  TwoHandedPowerMod,
+  MarksmanPowerMod,
+  BlockPowerMod,
+  SmithingPowerMod,
+  HeavyArmorPowerMod,
+  LightArmorPowerMod,
+  PickPocketPowerMod,
+  LockpickingPowerMod,
+  SneakPowerMod,
+  AlchemyPowerMod,
+  SpeechcraftPowerMod,
+  AlterationPowerMod,
+  ConjurationPowerMod,
+  DestructionPowerMod,
+  IllusionPowerMod,
+  RestorationPowerMod,
+  EnchantingPowerMod,
+  DragonRend,
+  AttackDamageMult,
+  HealRateMult_or_CombatHealthRegenMultMod,
+  MagickaRateMult_or_CombatHealthRegenMultPowerMod,
+  StaminaRateMult,
+  WerewolfPerks,
+  VampirePerks,
+  GrabActorOffset,
+  Grabbed,
+  DEPRECATED05,
+  ReflectDamage
+};
+
+class MGEF : public RecordHeader
+{
+public:
+  static constexpr auto kType = "MGEF";
+
+  struct DATA
+  {
+    // primary actor value
+    ActorValue primaryAV = espm::ActorValue::None;
+  };
+
+  struct Data
+  {
+    DATA data;
```

Comment:
Maybe it's better to get rid of this extra struct? Or is it needed for some consistency? Anyway, naming here kinda overlaps

Reply:
it wouldn't even compile